src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -101,7 +112,11 @@ p5.RendererGL.prototype.vertex = function(x, y) {
     v = arguments[4];
   }
   const vert = new p5.Vector(x, y, z);
-  this.immediateMode.geometry.vertices.push(vert);
+  if (this.immediateMode.isContour) {
+    this.immediateMode.geometry.contourVertices.push(vert);
```

Comment:
Nice work, I think this is generally the right approach! The one difference is that we might have multiple `beginContour`/`endContour` calls within one shape. So we probably need `contourVertices` to be an array of arrays in WebGL mode, instead of just a single array.

Reply:
Eventually, we'll need to pass all the contours into this array in `_tesselateShape` here: https://github.com/processing/p5.js/blob/393ddbc50457734cba8b2128aa20ec43fc62de7b/src/webgl/p5.RendererGL.Immediate.js#L333

This probably means we'll need to store `vertexNormals`, `vertexColors`, `vertexStrokeColors`, and `uvs` for each contour as well as just the position.